src/main/java/pl/anasz/cinema/repositories/ScreeningRepository.java

```diff
@@ -0,0 +1,24 @@
+package pl.anasz.cinema.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+import pl.anasz.cinema.enities.Screening;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface ScreeningRepository extends JpaRepository<Screening, Long> {
+
+      @Query("SELECT sc.roomId FROM Screening sc WHERE sc.movie.title = :movieId AND sc.screeningTime = :screeningTime")
+      Optional<Long> findRoomIdForScreening(@Param("movieId") String movieTitle, @Param("screeningTime")LocalDateTime screeningTime);
+
+      @Query("SELECT sc.movie.title,sc.screeningTime FROM Screening sc WHERE sc.screeningTime BETWEEN :date_1 AND :date_2 GROUP BY sc.movie.title,sc.screeningTime")
```

Comment:
Czemu robisz tutaj groupBy?

Reply:
Oczywiście to moja osobista pomyłka, gdyż powinnam użyć ORDER BY do posortowania wyników po poszczególnych kolumnach, czyli tytułu filmu i daty wyświetlenia, zgodnie z wymaganiami.